ios/ios_app_delegate.mm

```diff
@@ -7,10 +6,8 @@ @interface QIOSApplicationDelegate ()
 
 @implementation QIOSApplicationDelegate
 
-
-
-- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
-
+- (BOOL)application:(UIApplication*)application didFinishLaunchingWithOptions:(NSDictionary*)launchOptions
```

Comment:
See, it's lines like this where I feel clang-format hasn't done its job right.
```
- ( BOOL ) application: ( UIApplication* ) application didFinishLaunchingWithOptions: ( NSDictionary* ) launchOptions
```
but I don't  know Objective-C...

Reply:
Is that "too many" as in "won't compile" or as in "not normal"?

Reply:
Oh it's just purely a stylistic comment - there are _religious_ discussions about how to format Obj-C....
It makes no difference to compilation.